pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[tool.poetry]
+name = "flexmock"
+version = "0.11.0"
+description = "flexmock is a testing library for Python that makes it easy to create mocks, stubs and fakes."
+authors = ["Slavek Kabrda", "Herman Sheremetyev"]
+maintainers = [
+    "Adarsh Krishnan <adarshk7@gmail.com>",
+    "Christophe Riolo Uusivaara <riolo.christophe@gmail.com>",
+    "Olli Paakkunainen <olli@paakkunainen.fi>",
+]
+license = "BSD-2-Clause"
+repository = "https://github.com/flexmock/flexmock"
+documentation = "https://flexmock.readthedocs.io"
+readme = "README.md"
+keywords = ["mock", "testing", "test", "unittest", "pytest"]
+classifiers = [
+    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: Developers",
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Software Development :: Testing",
+]
+include = [
+    "CHANGELOG.md",
+    "docs/conf.py",
+    "docs/Makefile",
+    "docs/**/*.rst",
+    "LICENSE",
+    "README.md",
+    "src/**/*.py",
+    "tests/**/*.{py,sh}",
+]
+packages = [{ include = "flexmock", from = "src" }]
+
+[tool.poetry.dependencies]
+python = "^3.6.2"
+
+[tool.poetry.dev-dependencies]
+pytest = "^6.2.4"
+mypy = "^0.910"
+pylint = "^2.9.3"
+black = "^21.6b0"
+isort = "^5.9.1"
+Sphinx = "^4.0.3"
+tox = "^3.23.1"
+Twisted = "^21.2.0"
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.pylint.messages_control]
+disable = [
+    "too-many-instance-attributes",
+    "too-few-public-methods",
+    "logging-fstring-interpolation",
+    "too-many-function-args",
+    "too-many-arguments",
+    "bad-continuation",
+    "too-many-locals",
+    "too-many-branches",
+    "too-many-statements",
+    "too-many-public-methods",
+    "protected-access",
+    "broad-except",
+    "fixme",
```

Comment:
We could try to reduce those at some point, and use local disables, sparingly

Reply:
Yeah lets do some cleanup later. There were quite many places where pylint was complaining so I just disabled some rules globally. I will make an issue about this.

Reply:
https://github.com/flexmock/flexmock/issues/47